frontend/src/pages/projects/screens/spawner/storage/AddExistingStorageField.tsx

```diff
@@ -37,7 +37,7 @@ const AddExistingStorageField: React.FC<AddExistingStorageFieldProps> = ({
         ? storages.reduce((acc: TypeaheadSelectOption[], pvc) => {
             if (!existingStorageNames?.includes(pvc.metadata.name)) {
               acc.push({
-                value: pvc.metadata.name,
```

Comment:
this value should still the metadata name. only the content should change

Reply:
The `value` here is passed to  https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/pages/projects/screens/spawner/storage/AddExistingStorageField.tsx#L89, which further checks this condition https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/pages/projects/screens/detail/storage/utils.ts#L26

so using `getDisplayNameFromK8sResource() will return `display-name` if present or fallback to `metadata.name`
do you want to handle it other way??

Reply:
well we dont want to use display name for the value becuase it is not a key. it can be the same as other rows